projects/ion/src/lib/pagination/pagination.component.spec.ts

```diff
@@ -179,4 +197,47 @@ describe('Pagination > Events', () => {
     });
     expect(screen.queryAllByText('20 / página')).toHaveLength(1);
   });
+
+  const NUMBER_BUTTONS = [20, 30, 40, 46];
+
+  it.each(NUMBER_BUTTONS)(
```

Comment:
I think this assertion is redundant because you already check if items per page button is disabled above, it already ensures that no option will appear when this button is clicked.

Reply:
Yes I agree. When I was writing this test I thought so, but I chose to err on the side of more. But believe that the below is necessary because it validates that the operation remains normal in cases where the load is false.
Can I remove this test?

Reply:
Yes, you can remove this one.
About the assertion below, I don't know if it's necessary either because the 'Pagination > Events' tests already checks the behaviour when `loading` is falsy (false or undefined in this case will reproduce the same result), so, it is kinda redundant too.